pyctcdecode/language_model.py

```diff
@@ -32,6 +32,43 @@
     )
 
 
+def load_unigram_set_from_arpa(arpa_path: str) -> Set[str]:
```

Comment:
is there any way this could silently fail?  what if the arpa doesn't contain a 1-grams section?  should we raise if unigrams returns empty?

Reply:
technically i think min 2 grams are required for kenlm, so hopefully you'd just get a kenlm error, but can't help to add